Cargo.toml

```diff
@@ -45,8 +45,8 @@ rand                  = "^0.8"
 regex                 = { version = "^1.4", optional = true, default_features = false, features = ["std"] }
 reqwest               = { version = "0.11", features = ["json"] }
 rpassword             = { version = "^5.0", optional = true }
-rpki                  = { version = "0.13.1", features = [ "repository", "rrdp", "serde" ] }
-# rpki                  = { version = "0.13.1-rc1", git = "https://github.com/NLnetLabs/rpki-rs/", features = [ "repository", "rrdp", "serde" ] }
+rpki                  = { version = "0.13.2", features = [ "compat", "repository", "rrdp", "serde" ] }
+# rpki                  = { version = "0.13.2-dev", git = "https://github.com/NLnetLabs/rpki-rs/", branch = "compat-for-prefix-lens", features = [ "compat", "repository", "rrdp", "serde" ] }
```

Comment:
This line can be removed, right?

Reply:
I am keeping it for convenience whenever I need to depend on a development version. Is it in the way?

Reply:
It just offends my wonky painting sensibility ;-)